src/ts/file-reader.ts

```diff
@@ -1,9 +1,9 @@
 import { Har } from "har-format";
 
-declare const zip: any;
+const zipJs = () => (window["zip"] || {});
 
 // use zip
-zip.useWebWorkers = false;
+zipJs().useWebWorkers = false;
```

Comment:
In case `zip` gets loaded after the file-reader this would not be set, can you move this to getter?

Reply:
Sure! Thanks for the quick reply!